src/Analysis/Codelyzer.Analysis/CSharpCodeAnalyzer.cs

```diff
@@ -39,6 +39,18 @@ public override async Task<List<AnalyzerResult>> AnalyzeSolution(string solution
         {
             return await Analyze(solutionPath);
         }
+
+        ///<inheritdoc/>
+        public override async IAsyncEnumerable<AnalyzerResult> AnalyzeSolutionGenerator(string solutionPath)
```

Comment:
Method renaming suggestion:
1. Change method name to "AnalyzeSolutionWithGenerator" to make difference between this method and original "AnalyzeSolution" method more clear
2. Append "Async" since method is async